node/src/blockchain/blockchain_interface.rs

```diff
@@ -530,7 +530,7 @@ mod tests {
         let to = "0x3f69f9efd4f2592fd70be8c32ecd9dce71c472fc";
         let port = find_free_port();
 
-        let (tx, rx) = mpsc::sync_channel(1337);
+        let (tx, rx) = bounded(1337);
```

Comment:
This is just a guess, I might be wrong here.
This weird number meaning the number of messages to be together maximally inside the internal queue makes me wonder that it isn't ideal. (Somebody probably tried to pick some random number madly large so it doesn't reach its limits. Or I'm wrong and it works with bytes separately and this is number of bytes that we are willing to transport in this test maximally but I doubt it).
Please just try to change it to unbounded() crossbeam channel and try this single test immediately after. If it passes use it if not leave it as you have it now.

Reply:
No issue changing to unbounded.